tests/view_tests/tests/test_debug.py

```diff
@@ -260,9 +260,10 @@ def test_handle_db_exception(self):
 class NonDjangoTemplatesDebugViewTests(SimpleTestCase):
 
     def test_400(self):
-        # Ensure that when DEBUG=True, technical_500_template() is called.
-        response = self.client.get('/raises400/')
-        self.assertContains(response, '<div class="context" id="', status_code=400)
+        with patch_logger('django.security.SuspiciousOperation', 'error'):
+            # Ensure that when DEBUG=True, technical_500_template() is called.
```

Comment:
You could leave this where it is since it seems to be "docstringish", although in the spirit of https://github.com/django/django/pull/7439, the "Ensure that" preamble could be removed.